### PR TITLE
Edit color input, context. Edit implemented pages. Implement waiting room (host view only)

### PR DESCRIPTION
**Pull request by luangjay**
I forgot to format files...

### CODE REVIEW

File: src/components/pages/waitingRoom/index.js [hunk 1]
```diff
@@ -1,28 +1,315 @@
...
+    height: '8vh',
+    border: '1px solid black',
+    color: 'white',
+    backgroundColor: 'white',
+    [theme.breakpoints.down('sm')]: {
+      width: '5vh',
+      height: '5vh'
+    }
+  },
+  kickButton: {
```
Comment (PKhing) [inline src/components/pages/waitingRoom/index.js:120]: Kick button become white when hover?

**Review by PKhing — CHANGES_REQUESTED** [review]

File: src/components/pages/waitingRoom/index.js (hunk 1 above)
Comment (luangjay) [reply src/components/pages/waitingRoom/index.js:120]: Sorry I forgot

Comment (luangjay) [reply src/components/pages/waitingRoom/index.js:120]: Which color should it be?

Comment (PKhing) [reply src/components/pages/waitingRoom/index.js:120]: Dark grey?

Comment (luangjay) [reply src/components/pages/waitingRoom/index.js:120]: okay

**Review by PKhing — APPROVED** [review]

**Pull request merged**